### PR TITLE
[RPD-138] Update matcha repo readme with links to documentation

### PR DESCRIPTION
**Pull request by dudeperf3ct**
In this PR, we update the missing links to documentation for readme.

## Checklist

Please ensure you have done the following:

* [x] I have read the [CONTRIBUTING](https://github.com/fuzzylabs/matcha/blob/main/CONTRIBUTING.md) guide.
* [x] I have updated the documentation if required.
* [ ] I have added tests which cover my changes.

## Type of change

Tick all those that apply:

* [ ] Bug Fix (non-breaking change, fixing an issue)
* [x] New feature (non-breaking change to add functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Other (add details above)

### CODE REVIEW

File: README.md
Comment (Christopher-Norman) [inline README.md:1]: Could we change:
```
The current version of `matcha` provisions the above with the single command, using sensible defaults for the infrastructure.

There are more components that could be added here, for example, managing and versioning datasets, the management of models (registration and governance), and monitoring models. Adding all of these is on our roadmap.
```
to
```
The current version of `matcha` provisions the above with a single command, using sensible defaults for the infrastructure.

More components could be added here, for example, managing and versioning datasets, managing models (registration and governance), and monitoring models. Adding all of these is on our roadmap.
```

**Review by Christopher-Norman — CHANGES_REQUESTED** [review]
LGTM, just a couple changes in grammar while we are working on the documentation would be good.

**Review by Christopher-Norman — APPROVED** [review]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 `matcha` is a tool for provisioning MLOps environments to the cloud.
 
+Jump to [get started](https://fuzzylabs.github.io/matcha/getting-started/) section to help you get started using matcha quickly.
```
Comment (JonoCX) [inline docs/index.md:3]: I'd remove the "quickly" here as per https://justsimply.dev/

**Review by JonoCX — CHANGES_REQUESTED** [review]
Nice - see comments

**Review by JonoCX — APPROVED** [review]
LGTM 🚀

**Pull request merged**